templates/membership/register.html.twig

```diff
@@ -45,13 +44,14 @@
 
         <section class="register__content">
             <article class="l__wrapper--slim b__nudge--bottom-60">
-                <p class="text--body">En Marche ! est un collectif citoyen qui part d'une conviction :
-                    les Français doivent être au cœur de la vie politique et non son décor. Ce collectif
-                    n'est pas l'addition d'individus, mais le rassemblement de personnes engagées qui
-                    partagent un socle de valeurs.</p>
+                <p class="text--body">Vous avez dès maintenant la possibilité d'adhérer à En Marche.
+                Nous sommes un collectif citoyen qui part d'une conviction : les Français doivent être au cœur
+                de la vie politique et non son décor.<br/>
+                Pour ce faire, veuillez remplir ces informations :</p>
             </article>
             <article class="l__wrapper--slim b__nudge--bottom-huge l__col l__col--center">
                 <div class="register__form">
+                    {% set isCreateForm = true %}
```

Comment:
Is this var used? should be named `is_create_form`

Reply:
fixed